src/net_processing.cpp

```diff
@@ -3261,6 +3265,25 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
             if (fRelay) pfrom.m_relays_txs = true;
         }
 
+        if (greatest_common_version >= WTXID_RELAY_VERSION && m_txreconciliation) {
```

Comment:
Currently no. Do you see any benefit in doing so?

Reply:
Is this still the case? At least btcd will not require a protocol bump, reading https://github.com/btcsuite/btcd/blob/38ee9a41c8f8aa24a079a28f5e8a86faecffdfe1/wire/protocol.go#L57-L59 . Are there any others?

Reply:
I am low-key leaning towards not bumping the version following the BIP155 path. This PR got several acks already, which also means some sort of (at least soft) approval from those reviewers. Might be not a big deal, but I think I already went 3 times back-and-forth with this issue.

I will keep this thread open for those willing to insist on the version bump. @sipa or anyone please let me know if not bumping prevents you from acking. 

This can also be changed in the future, anyone can open a PR to the repo (or the BIP).

Reply:
How about we actually document the messages-before-verack idea as a standard then? Here's a draft https://github.com/ajtowns/bips/blob/202210-p2pfeatures/bip-p2pfeatures.mediawiki

Reply:
Looks good, I think it's worth opening a PR to the bips repo?

Reply:
Sure; https://github.com/bitcoin/bips/pull/1377